src/util.js

```diff
@@ -1,3 +1,312 @@
+// PathData object is based upon `Source`, here https://github.com/jarek-foksa/path-data-polyfill.js/blob/master/path-data-polyfill.js
```

Comment:
The nature of the changes in this file strike me as a polyfill. I'm not an active contributor to this project, but as an interested consumer, I would think it might make more sense if this were moved to src/polyfill.js.

For what it's worth, I favour this PR as a whole over #1564, as we should be providing forwardbridges to current APIs, not backbridges to legacy ones.